python-sdk/tests/sql/operators/test_dataframe.py

```diff
@@ -334,3 +334,43 @@ def sample_df_1(**kwargs):
 
     task = sample_df_1(output_table=output_table)
     assert task.operator.outlets == []
+
```

Comment:
Deletion of `tests/airflow/__init__.py` file looks like a mistake ?

Reply:
@kaxil we need to figure out how to handle that, because it's causing all kinds of issues in pycharm. Basically airflow imports don't work because the ide thinks we're importing the tests airflow package instead of the apache airflow package.

Reply:
I would like to see that when you are back. I can't replicate it on my Pycharm